dolphinscheduler-dist/src/main/assembly/create-symlinks.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+script_dir=`dirname $0`
+cd $script_dir/../../../target
+package_file=`ls apache-dolphinscheduler-*-bin.tar.gz`
+echo $package_file
+decompress_dirname="${package_file%.tar.gz}"
+rm -rf $decompress_dirname
+#Decompress package file
+tar -xf $package_file
+cd $decompress_dirname
+
+SHARED_LIB_DIR="libs"
+# create share lib directory
+mkdir -p $SHARED_LIB_DIR
+
+echo 'iterate through the lib directory for all subprojects'
+for module in api-server master-server worker-server alert-server tools; do
+  MODULE_LIB_DIR="$module/libs"
+  echo "handling $MODULE_LIB_DIR"
+
+  if [ -d "$MODULE_LIB_DIR" ]; then
+    cd $MODULE_LIB_DIR
+
+    for jar in `ls *.jar`; do
+      # Move jar file to share lib directory
+      mv $jar ../../$SHARED_LIB_DIR/
+
+      # Create a symbolic link in the subproject's lib directory
+      ln -s ../../$SHARED_LIB_DIR/$jar .
```

Comment:
This way is not compatible with k8s.

Reply:
why? your mean k8s pod not support symbolic link? @SbloodyS

Reply:
DS docker image does not have other server's file. @zhuxt2015

Reply:
If this script is just used for publishing binary packages, it will not impact the deployment of k8s.  During the Docker image build process, the contents of the target directory for each module are directly copied. @SbloodyS

Reply:
We can get CI to test if it can work. Please fix CI @zhuxt2015

Reply:
Done